Bio/SeqFeature.py

```diff
@@ -1359,30 +1376,33 @@ class ExactPosition(int, AbstractPosition):
     """
 
     def __new__(cls, position, extension=0):
+        """Create an ExactPosition object."""
         if extension != 0:
             raise AttributeError("Non-zero extension %s for exact position."
                                  % extension)
         return int.__new__(cls, position)
 
     def __repr__(self):
-        """String representation of the ExactPosition location for debugging."""
+        """Represent the ExactPosition object as a string for debugging."""
         return "%s(%i)" % (self.__class__.__name__, int(self))
 
     @property
     def position(self):
-        """Legacy attribute to get position as integer (OBSOLETE)."""
+        """Legacy attribute to get location as integer (OBSOLETE)."""
```

Comment:
I think this should be position (consistent with how position and location are used here and in the Tutorial).